### ISSUE TITLE
several questions when using mmsegmentation

### ISSUE THREAD

**Issue by wwjwy** [open-mmlab/mmsegmentation#806]
1
When I use mmseg, I noticed that all the default loss functions use the cross-entropy loss function instead of dice loss. What are the considerations? Generally speaking, don't most segmentation tasks use dice loss?

2
I have finished my training using the "hooks=[dict(type='TextLoggerHook', by_epoch=False)''. Now, I want to compare the loss of different models in the same Figure. What should I do?

Thank you very much!

**Comment by MengzhangLI** [comment]
Hi, @wwjwy 

1. 
(1) Because in previous studies, especially in natural image semantic segmentation tasks/ datasets, loss function was not been studied very well (to some degree, it is not a hot topic in this area). Thus, our default settings just foucs on model itself and ohter configurations discussed in most paper. 
(2) We are doing some works about loss functions, different combinations of them. Maybe our works and codes would be released in the couple of months.
(3) Dice loss is very important especially in medical image segmentation, but in natural image dataset such as Cityscapes or ADE20K, default settings usually use Cross Entropy Loss. If you want to use it, you can change `loss_decode` in configuration to meet up your own needs.

2.
I'm not pretty sure about your question, but I try to answer you:
(1) set fixed seed during training, for instance: `--seed 0` to make relatively fair comparison.
(2) In many paper, they usually control model or loss functions at least one side is the same. For example, this [repositories](https://github.com/JunMa11/SegLoss/tree/master/test) enables the model and the dataset identical. So I think different models in the same figure should have same loss functions.

Best,

**Comment by wwjwy** [comment]
> Hi, @wwjwy
> 
> 1. (1) Because in previous studies, especially in natural image semantic segmentation tasks/ datasets, loss function was not been studied very well (to some degree, it is not a hot topic in this area). Thus, our default settings just foucs on model itself and ohter configurations discussed in most paper.
>    (2) We are doing some works about loss functions, different combinations of them. Maybe our works and codes would be released in the couple of months.
>    (3) Dice loss is very important especially in medical image segmentation, but in natural image dataset such as Cityscapes or ADE20K, default settings usually use Cross Entropy Loss. If you want to use it, you can change `loss_decode` in configuration to meet up your own needs.
> 2. I'm not pretty sure about your question, but I try to answer you:
>    (1) set fixed seed during training, for instance: `--seed 0` to make relatively fair comparison.
>    (2) In many paper, they usually control model or loss functions at least one side is the same. For example, this [repositories](https://github.com/JunMa11/SegLoss/tree/master/test) enables the model and the dataset identical. So I think different models in the same figure should have same loss functions.
> 
> Best,

Thank you very much for your timely response!
